.github/workflows/ci.yaml

```diff
@@ -12,6 +12,38 @@ on:
       - synchronize
 
 jobs:
+  define-scanner-job-matrix:
+    outputs:
+      matrix: ${{ steps.define-scanner-job-matrix.outputs.matrix }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Define the matrix for build jobs
+      id: define-scanner-job-matrix
+      run: |
+        source './scripts/ci/lib.sh'
+
+        # If goarch is updated, be sure to update architectures in push-manifests below.
+        matrix='{ "build_and_push": { "name":["default"], "goos":["linux"], "goarch":["amd64", "arm64", "ppc64le", "s390x"] }, "push_manifests": { "name":["default"] } }'
```

Comment:
no need for `"name"` since we don't do `prerelease` builds in this repo

Reply:
Yes, stil working on it.  Thanks for taking a quick look will add these changes on next checkin.  Do we need "push_manifests"?  don't see a reference to it in this repo either...
```
        matrix='{ "build_and_push": {"goos":["linux"], "goarch":["amd64", "arm64", "ppc64le", "s390x"] }, "push_manifests": { "name":["default"] } }'
```